core/templates/components/skill-selector/skill-selector.component.html

```diff
@@ -0,0 +1,105 @@
+<div class="skill-selector-parent skill-container oppia-skill-container protractor-test-skill-container">
+  <div class="list-group oppia-list-group">
+    <mat-card layout="row" class="list-view-item protractor-test-skill-select-header">
+      Search By Skill
+      <input type="text" class="form-control protractor-test-skill-name-input" placeholder="Enter a skill name" ng-model="skillFilterText" autofocus>
+    </mat-card>
+    <div class="skill-list-box">
+      <mat-radio-group ng-model="selectedSkill" ng-change="setSelectedSkillId()">
+        <mat-card *ngFor="(topicName, subtopics) in categorizedSkills" class="list-view-item" *ngIf="checkIfTopicIsEmpty(topicName)">
+          <mat-card layout="row" class="list-view-item">
+            <h2><label class="oppia-topic-name-text">{{ topicName }}</label></h2>
+          </mat-card>
+          <div *ngIf="!checkIfEmpty(skills)" *ngFor="(subtopicName, skills) in subtopics" class="list-view-item ">
+            <mat-card class="list-view-item">
+              <label class="oppia-subtopic-name-text">{{ subtopicName }}</label>
+              <md-radio-button class="oppia-skills-list-item protractor-test-skills-list-item" *ngFor="skill in skills | filter:skillFilterText" ng-value="skill.id" aria-label="{{ skill.description }}">
+                {{ skill.description }}
+              </md-radio-button>
+            </mat-card>
+          </div>
+        </mat-card>
+        <mat-card *ngIf="untriagedSkillSummaries.length" class="list-view-item">
+          <mat-card layout="row" class="list-view-item">
+            <h2><label class="oppia-heading-text">Untriaged Skills</label></h2>
+          </mat-card>
+          <div *ngIf="!checkIfEmpty(untriagedSkillSummaries)" class="list-view-item ">
+            <mat-card class="list-view-item">
+              <md-radio-button class="oppia-skills-list-item protractor-test-skills-list-item" *ngFor="skill in untriagedSkillSummaries | filter:skillFilterText" ng-value="skill.id" aria-label="{{ skill.description }}">
+                {{ skill.description }}
+              </md-radio-button>
+            </mat-card>
+          </div>
+        </mat-card>
+      </mat-radio-group>
+    </div>
+  </div>
+  <div class="list-group oppia-filter-list-group">
+    <mat-card *ngIf="canAllowSkillsFromOtherTopics()" layout="row" class="list-view-item oppia-list-view-item">
+      <label>Filter By Topics</label>
+    </mat-card>
+    <div *ngIf="canAllowSkillsFromOtherTopics()" class="oppia-list-view-item-container">
+      <mat-card class="list-view-item ">
+        <md-checkbox class="oppia-skills-list-item" *ngFor="topic in topicFilterList" ng-model="topic.checked" ng-change="updateSkillsListOnTopicFilterChange()">{{ topic.topicName }}</md-checkbox>
```

Comment:
This should change to mat-checkbox, also check for ng-model and ng-change, ng-model -> [(ngModel)], ng-change -> (change).

Reply:
Done